apps/README.md

```diff
@@ -38,37 +35,36 @@ have to inform the TinyAra build system of that location.  There are several
 ways to do that:
 
 1) You can define CONFIG_APPS_DIR to be the full path to your application
-   directory in the TinyAra configuration file.
-2) You can provide the path to the application directory on the command line
-   like:  make APPDIR=<path> or make CONFIG_APPS_DIR=<path>
+   directory in the TinyAra configuration file.  
+2) You can provide the path to the application directory on the command line  
+   like:  make APPDIR=\<path\> or make CONFIG_APPS_DIR=\<path\>
 3) When you configure TinyAra using tools/configure.sh, you can provide that
-   path to the application directory on the configuration command line
-   like: ./configure.sh -a <app-dir> <board-name>/<config-name>
+   path to the application directory on the configuration command line  
+   like: ./configure.sh -a \<app-dir\> \<board-name\>/\<config-name\>
+
+## Application Configuration File
 
-Application Configuration File
-------------------------------
 The TinyAra configuration uses kconfig-frontends tools and the TinyAra
 configuration file (.config) file.  For example, the TinyAra .config
 may have:
 
-  CONFIG_EXAMPLES_HELLO=y
+	CONFIG_EXAMPLES_HELLO=y
 
 This will select the apps/examples/hello in the following way:
 
-- The top-level make will include examples/Make.defs
-- examples/Make.defs will set CONFIGURED_APPS += examples/hello
+- The top-level make will include examples/Make.defs  
+- examples/Make.defs will set CONFIGURED_APPS += examples/hello  
   like this:
 
-  ifeq ($(CONFIG_EXAMPLES_HELLO),y)
-  CONFIGURED_APPS += examples/hello
-  endif
+	ifeq ($(CONFIG_EXAMPLES_HELLO),y)  
+	CONFIGURED_APPS += examples/hello  
+	endif
 
 
-Building TinyAra with Board-Specific Pieces Outside the Source Tree
------------------------------------------------------------------
+## Building TinyAra with Board-Specific Pieces Outside the Source Tree
```

Comment:
In my opinion we can rename the heading as FAQ meaning frequently asked questions. We can add more clarifications/guides for the apps directory in this section.